test/Api.IntegrationTest/Factories/ApiApplicationFactory.cs

```diff
@@ -2,17 +2,24 @@
 using Bit.IntegrationTestCommon.Factories;
 using IdentityServer4.AccessTokenValidation;
 using Microsoft.AspNetCore.TestHost;
+using Microsoft.Data.Sqlite;
 
 namespace Bit.Api.IntegrationTest.Factories;
 
 public class ApiApplicationFactory : WebApplicationFactoryBase<Startup>
 {
     private readonly IdentityApplicationFactory _identityApplicationFactory;
+    private readonly SqliteConnection _connection;
+    private const string _connectionString = "DataSource=:memory:";
 
     public ApiApplicationFactory()
     {
+        _connection = new SqliteConnection(_connectionString);
+        _connection.Open();
```

Comment:
**nitpick (non-blocking)**: You could skip the whole `_connection` member and assign directly to `SqliteConnection`.

```c#
SqliteConnection = new SqliteConnection(_connectionString);
SqliteConnection.Open();

_identityApplicationFactory = new IdentityApplicationFactory();
_identityApplicationFactory.SqliteConnection = SqliteConnection;
```

Reply:
Seems worth making the change now vs later. 

https://github.com/bitwarden/server/pull/3292/commits/6048b774dd571118b1d85a3f5e156eb9928cbc23